gap/ClassicalMaximals.gi

```diff
@@ -827,3 +827,28 @@ function(n, q, classes...)
 
     return maximalSubgroups;
 end);
+
+
+BindGlobal("C2SubgroupsSymplecticGroupGeneric",
+function(n, q)
+    local result, divisorListOfn, t;
+    
+    result := [];
+
+    divisorListOfn := List(DivisorsInt(n));
+    Remove(divisorListOfn, 1);
+
+    # type Sp(m, q) \wr Sym(t) subgroups
+    for t in divisorListOfn do
+        if IsEvenInt(QuoInt(n, t)) then
+            Add(result, SpNonDegenerateImprimitives(n, q, t));
+        fi;
+    od;
```

Comment:
https://github.com/gap-packages/ClassicalMaximals/pull/76/files#r746644581 seems to apply here?